src/Libraries/DSIronPython/DSIronPython.csproj

```diff
@@ -36,6 +36,10 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="ICSharpCode.AvalonEdit">
+      <HintPath>..\..\..\extern\avalonEdit\ICSharpCode.AvalonEdit.dll</HintPath>
```

Comment:
Can we use an AvalonEdit NuGet package instead?

Reply:
What version we are using? Nuget.org shows they only published 4.0 +

Reply:
I just checked, seems we are using file version 4.3.1.9429 which is not an official version from the list https://www.nuget.org/packages/AvalonEdit/. I encourage us move on to use nuget dependencies.

Reply:
I actually have addressed similar issue for fontawesome in my all-you-can-eat-buffet branch. This kind of dech_debt is quite easy. If you guys are OK, I will log a Jira to cover all of these. FYI: @DynamoDS/dynamo

Reply:
Done. https://jira.autodesk.com/browse/DYN-2639

Reply:
Thanks Aaron.